packages/core/src/components/Label/LabelCelebrationAnimation.module.scss

```diff
@@ -0,0 +1,91 @@
+.celebration {
+  // Fallback perimeter, changes according to the path length
+  --container-perimeter: 840;
+  position: relative;
+
+  .svg {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+
+    .stroke {
+      fill: none;
+      stroke-width: 1;
+      stroke-linecap: round;
+      stroke-linejoin: round;
+      animation: stroke-rotate cubic-bezier(0.33, 0, 0.67, 1) forwards;
+      stroke-dasharray: var(--container-perimeter);
+      stroke-dashoffset: var(--container-perimeter);
+
+      &.base {
+        stroke: #ffcc00;
```

Comment:
Need to discuss colors with designers as they are legacy colors, will be replaced with variables

Reply:
After we will change all the colors let's QA how it looks in all the different products themes :)